packages/patternfly-4/react-charts/src/components/ChartDonut/examples/ChartDonut.md

```diff
@@ -33,15 +35,15 @@ import { ChartDonut } from '@patternfly/react-charts';
       title="100"
     />
   </div>
-</div>
+)
 ```
 
-```js title=Donut-chart-with-right-aligned-legend
+```js title=Right--aligned-legend
```

Comment:
It should be "Right-aligned" -- no dash. The title was "Donut chart with right aligned legend"